HTAN.model.csv

```diff
@@ -1030,3 +1032,8 @@ Barretts Esophagus Goblet Cells Present,Presence or absennce of Barretts esophag
 Pancreatitis Onset Year,Date of onset of pancreatitis.,,,,FALSE,Follow Up,,,num
 HTAN Parent Channel Metadata ID,HTAN ID for a level 3 channels table.,,,,TRUE, Imaging Level 4,,,
 Single Nucleus Capture,Nuclei isolation method,"Plates, 10x, droplet",,,FALSE,scmC-seq Level 1,,,
+Microarray Protocol Auxiliary File,"Auxiliary file describing the experimental protocols used, as described in the NCBI GEO microarray template, recorded as synapse ID (syn12345).",,"Component, Filename, File Format, HTAN Parent Data File ID, HTAN Data File ID",,FALSE,Assay,Microarray Level 1,,

```

Comment:
Check that this has the right number of columns and quotes. GH actions throwing an error